wpimath/src/test/native/cpp/geometry/Translation2dTest.cpp

```diff
@@ -38,6 +38,15 @@ TEST(Translation2dTest, RotateBy) {
   EXPECT_DOUBLE_EQ(3.0, rotated.Y().value());
 }
 
+TEST(Translation2dTest, RotateAround) {
+  const Translation2d another{2_m, 1_m};
+  const Translation2d other{3_m, 2_m};
+  const auto rotated = another.RotateAround(other, 180_deg);
+
+  EXPECT_NEAR(4.0, rotated.X().value(), 1e-9);
+  EXPECT_DOUBLE_EQ(3.0, rotated.Y().value());
```

Comment:
```suggestion
  EXPECT_NEAR(4.0, rotated.X().value(), 1e-9);
  EXPECT_NEAR(3.0, rotated.Y().value(), 1e-9);
```
Better to be consistent.

Reply:
I saw that in the RotateBy test above it used EXPECT_NEAR and EXPECT_DOUBLE_EQ and copied that, although looking at it now, I'm not too sure the reason why they are different - should I change both? Or just change RotateAround?

Reply:
RotateBy should be changed too, and we might as well do it in this PR too.